### ISSUE TITLE
Update issue to 2.4.134 from 2.4.132  Error: [MissingTableException] Table auth_keys for model AuthKey was not found in datasource default.

### ISSUE THREAD

**Issue by faustus25** [MISP/MISP#6582]
Performed an update for MISP from v2.4.132 to v2.4.134

Server Details:
Server version: 10.0.38-MariaDB-0ubuntu0.16.04.1 Ubuntu 16.04

Maria DB Version:
MariaDB [misp]> SELECT * FROM admin_settings WHERE setting='db_version';
+----+------------+-------+
| id | setting    | value |
+----+------------+-------+
|  1 | db_version | 59    |

Update commands:
cd /var/www/MISP
git pull origin 2.4
git submodule update --init --recursive --force
find /var/www/MISP -type d -exec chmod g=rx {} \;
chmod -R g+r,o= /var/www/MISP/
chown -R www-data:www-data /var/www/MISP/

Error:
2020-11-13 17:08:17 Error: [MissingTableException] Table auth_keys for model AuthKey was not found in datasource default.
Exception Attributes: array (
  'table' => 'auth_keys',
  'class' => 'AuthKey',
  'ds' => 'default',
)
Request URL: /users/login
Stack Trace:
#0 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(3686): Model->setSource('auth_keys')
#1 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(1394): Model->getDataSource()
#2 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(2663): Model->schema()
#3 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(1886): DboSource->fields(Object(AuthKey), 'AuthKey', '')
#4 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(1377): DboSource->generateAssociationQuery(Object(User), Object(AuthKey), 'hasMany', 'AuthKey', Array, Array, true)
#5 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Datasource/DboSource.php(1257): DboSource->queryAssociation(Object(User), Object(AuthKey), 'hasMany', 'AuthKey', Array, Array, true, Array, 0, Array)
#6 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(3053): DboSource->read(Object(User), Array)
#7 /var/www/MISP/app/Lib/cakephp/lib/Cake/Model/Model.php(3025): Model->_readDataSource('count', Array)
#8 /var/www/MISP/app/Controller/UsersController.php(1247): Model->find('count')
#9 [internal function]: UsersController->login()
#10 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Controller.php(499): ReflectionMethod->invokeArgs(Object(UsersController), Array)
#11 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(193): Controller->invokeAction(Object(CakeRequest))
#12 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(UsersController), Object(CakeRequest))
#13 /var/www/MISP/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#14 {main}

Unable to load login page for MISP now.

What  can I do to correct this?

**Comment by mokaddem** [comment]
It seems your MISP instance cannot start the database update.
It could be that your workers are stuck or dead, if that's the case, running
```bash
./cake Admin runUpdates
```
Might solve the issue. Don't forget to restart any dead workers afterward

**Comment by faustus25** [comment]
> 
> 
> It seems your MISP instance cannot start the database update.
> It could be that your workers are stuck or dead, if that's the case, running
> 
> ```shell
> ./cake Admin runUpdates
> ```
> 
> Might solve the issue. Don't forget to restart any dead workers afterward

That worked @mokaddem , good catch!

```
misp@misp:/var/www/MISP/app/Console# sudo -u www-data ./cake Admin runUpdates

Welcome to CakePHP v2.10.22 Console
---------------------------------------------------------------
App : app
Path: /var/www/MISP/app/
---------------------------------------------------------------
Executing all updates to bring the database up to date with the current version.
Executing 60..................Done
Executing 61..................Done
All updates completed.
```

Checked the workers too and they are all running as expected.

Interestingly had the issue on both 2 MISP installed Ubuntu servers, 16.04 and 18.04 and the issue resolved on 16.04 eventually whereas the 18.04 the issue persisted.

**Comment by mokaddem** [comment]
Cool! Thanks for letting us know!
